src/components/ManagePage.tsx

```diff
@@ -1,28 +1,26 @@
-import { atom, useRecoilValue } from "recoil";
 import { Container, Flex } from "@mantine/core";
 import { ReactElement } from "react";
 import { CustomAccordion } from "components/CustomAccordion/CustomAccordion";
 import { StreamAccordionItem } from "components/StreamItemAccordion/StreamAccordionItem";
 import { TextLg } from "components/TextVariants";
 import { Stream, useReceiverStreams, useSenderStreams } from "hooks/Streams";
-import {
-  SendingAndReceiving,
-  sendingOrReceivingAtom,
-} from "components/SendingAndRecieving/SendingAndReceiving";
+import { SendingAndReceiving } from "components/SendingAndRecieving/SendingAndReceiving";
 import { useIsMobile } from "hooks/useIsMobile";
 import { isEmpty } from "lodash";
-
-export const globalStreams = atom({
-  key: "globalStreams",
-  default: [] as Stream[],
-});
+import { useSelector } from "react-redux";
+import { RootState } from "@/redux/store";
 
 export const ManagePage = () => {
-  const streams = useRecoilValue(globalStreams);
+  const streams = useSelector(
```

Comment:
Looking at the useage guide, shouldn't we use the getSelectors adapter to make this easier?

https://redux-toolkit.js.org/usage/usage-guide